### ISSUE TITLE
Error {"status":"error","code":404,"message":"the hostname of the origin is unresolvable (dns) or blocked by policy."}

### ISSUE THREAD

**Issue by vetp** [andrieslouw/imagesweserv#415]
{"status":"error","code":404,"message":"the hostname of the origin is unresolvable (dns) or blocked by policy."}

**Comment by kleisauke** [comment]
Please can you provide a sample URL that allows someone else to reproduce. Note that we block questionable categories on our public service due to legal concerns. See: https://github.com/weserv/images/issues/285#issuecomment-828176654.

**Comment by onionlian** [comment]
Hi:

my websit https://www.cool-style.com.tw/wd2/ also has same error
cdn-origin.cool-style.com.tw

weserv url https://images.weserv.nl/?w=800&h=424&fit=cover&url=https%3A%2F%2Fcdn-origin.cool-style.com.tw%2Fcool%2F2024%2F05%2Fhonda-cover.png

origin https://cdn-origin.cool-style.com.tw/cool/2024/05/honda-cover.png

thanks

**Comment by kleisauke** [comment]
@vetp I'll close due to inactivity but please feel free to reopen with more details if further help is required.

@onionlian Please open a new issue if you are still having problems.
